src/pages/_app.js

```diff
@@ -123,9 +115,17 @@ ApplicationLayout.propTypes = {
 const Layout = ({ children }) => {
   const { asPath } = useRouter();
 
-  if (asPath.startsWith('/login')) return <LoginLayout>{children}</LoginLayout>;
-  if (asPath.startsWith('/404')) return children;
-  return <ApplicationLayout>{children}</ApplicationLayout>;
+  if (
+    asPath.startsWith('/login') ||
+    asPath.startsWith('/404') ||
+    asPath.startsWith('/error')
```

Comment:
```suggestion
    asPath.startsWith('/404')
```

Reply:
@geoffreydhuyvetters Can you change this? the error page doesn't exist anymore so

Reply:
FYI you can also explicitly request changes when reviewing, so merging becomes impossible until it's fixed 😄